dev/PushNotifications/PushNotificationReceivedEventArgs.cpp

```diff
@@ -80,7 +77,7 @@ namespace winrt::Microsoft::Windows::PushNotifications::implementation
 
     void PushNotificationReceivedEventArgs::Canceled(winrt::event_token const& token) noexcept
     {
-        if (m_isBIAvailable)
+        if (m_isUnpackagedApp)
```

Comment:
Wrong condition. You want:
```suggestion
        if (!m_isUnpackagedApp)
```